app.js

```diff
@@ -73,17 +73,17 @@ IMPORTANT DETAIL: You may not use the arithmetic operator + in this function. To
 Test this function by hand in the console to get it working, and when you think it is finished, uncomment the call for the testSumArray() function and see if the test passes.*/
 
 // Write your code here
-var testArray = [2,3,4]; //eslint-disable-line
+var testArray = [2,3,4,5]; //eslint-disable-line
 
 function sumArray(testArray){ //eslint-disable-line
   var sumArrayValues = [];
-  for (var i = 0; i < (testArray.length - 2); i++){
-    console.log(i);
-    sumArrayValues[0] = sum(testArray[i], sum(testArray[(i + 1)], testArray[(i + 2)])[0])[0];
-    console.log(sumArrayValues[0]);
+  sumArrayValues[0] = 0;
```

Comment:
FWIW, I would probably use a separate variable here (`sum`? `sumResult`, since `sum` is a function?) instead of setting/getting `sumArrayValues[0]` throughout. Just makes it a tiny bit harder to read.